src/pages/homepage/HomePage.js

```diff
@@ -0,0 +1,91 @@
+import styled from "styled-components";
+import place from "./images/place.png";
+import volenteer from "./images/voloenteer.png";
+// import { useState } from "react";
+import useAuthContext from "./hooks/useAuthContext";
+// Styled Components
+const Header = styled.header`
```

Comment:
best practice when u use styled components is to put it in seperated file with appropriate name ,
for example :
Homepage.js -> includes jsx and logic
Homepage.style.js -> include the styling